core/internal/testutils/configtest/general_config.go

```diff
@@ -66,9 +66,13 @@ func overrides(c *chainlink.Config, s *chainlink.Secrets) {
 	c.WebServer.TLS.ListenIP = &testIP
 
 	chainID := big.NewI(evmclient.NullClientChainID)
+
+	chainCfg := evmcfg.Defaults(chainID)
+	chainCfg.LogPollInterval = commonconfig.MustNewDuration(1 * time.Second) // speed it up from the standard 15s for tests
```

Comment:
Seems reasonable, but how does it affect other tests? Presumably nothing broke - but does this also represent an opportunity to speed up any tests by tuning related values?